### ISSUE TITLE
bug: After navigating more than three pages, fast click the back button and the page will become blank

### ISSUE THREAD

**Issue by qq124719010** [ionic-team/ionic-framework#22430]
**Ionic version:**
[ ] 4.x
[x] 5.x

**Current behavior:**
When navigating from page A to page B, then from page B to page C, and then from page C to page D, and finally, fast click the back button many times, the page will become blank

**Expected behavior:**
The page should never be blank.

**Steps to reproduce:**

**pageA:** 
```
<ion-header>
  <ion-toolbar> 
    <ion-title> PageA </ion-title> 
  </ion-toolbar>
</ion-header>
<ion-content>
    <a routerLink="/pageB">goto pageB</a>
</ion-content>
```

**pageB:** 
```
<ion-header>
  <ion-toolbar>
    <ion-buttons slot="start">
      <ion-back-button></ion-back-button>
    </ion-buttons>
    <ion-title> PageB </ion-title> 
  </ion-toolbar>
</ion-header>
<ion-content>
    <a routerLink="/pageC">goto pageC</a>
</ion-content>
```
 
**pageC:** 
```
<ion-header>
  <ion-toolbar>
    <ion-buttons slot="start">
      <ion-back-button></ion-back-button>
    </ion-buttons>
    <ion-title> PageC </ion-title> 
  </ion-toolbar>
</ion-header>
<ion-content>
    <a routerLink="/pageD">goto pageD</a>
</ion-content>
```
 
 
**pageD:**  
```
<ion-header>
  <ion-toolbar>
    <ion-buttons slot="start">
      <ion-back-button></ion-back-button>
    </ion-buttons>
    <ion-title> PageD </ion-title> 
  </ion-toolbar>
</ion-header>
<ion-content>
    <p>PageD</p>
</ion-content>
```

**Comment by liamdebeasi** [comment]
Thanks for the issue. Can you please reproduce this in an Ionic starter app and provide a link to the GitHub repo?

**Comment by qq124719010** [comment]
I found that this problem has been fixed in the latest version 5.4.2，Thank you!
